Documentation/gettingstarted/k8s-install-helm.rst

```diff
@@ -81,9 +81,24 @@ Install Cilium
        * Reconfigure kubelet to run in CNI mode
        * Mount the eBPF filesystem
 
-    .. group-tab:: AKS
+    .. group-tab:: AKS (BYOCNI)
 
-       .. include:: requirements-aks.rst
+       .. include:: requirements-aks-byocni.rst
+
+       **Install Cilium:**
+
+       Deploy Cilium release via Helm:
+
+       .. parsed-literal::
+
+          helm install cilium |CHART_RELEASE| \\
+            --namespace kube-system \\
+            --set aksbyocni.enabled=true \\
+            --set nodeinit.enabled=true
```

Comment:
:100:

This is surprisingly simpler than I thought. Just test it out, working as expected 

```
$ helm install cilium ./cilium \
  --namespace kube-system \
  --set aksbyocni.enabled=true \
  --set nodeinit.enabled=true
NAME: cilium
LAST DEPLOYED: Sat Jun 18 23:16:18 2022
NAMESPACE: kube-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
You have successfully installed Cilium with Hubble.

Your release version is 1.11.90.

For any further help, visit https://docs.cilium.io/en/v1.11/gettinghelp

$ cilium status
    /¯¯\
 /¯¯\__/¯¯\    Cilium:         OK
 \__/¯¯\__/    Operator:       OK
 /¯¯\__/¯¯\    Hubble:         disabled
 \__/¯¯\__/    ClusterMesh:    disabled
    \__/

DaemonSet         cilium             Desired: 3, Ready: 3/3, Available: 3/3
Deployment        cilium-operator    Desired: 2, Ready: 2/2, Available: 2/2
Containers:       cilium             Running: 3
                  cilium-operator    Running: 2
Cluster Pods:     4/4 managed by Cilium
Image versions    cilium             quay.io/cilium/cilium-ci:latest: 3
                  cilium-operator    quay.io/cilium/operator-generic-ci:latest: 2

```